.eslintrc.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   "env": {
     "jest": true,
     "browser": true,
+    "jquery": true
   },
+  "plugins": [
+    "dollar-sign",

```

Comment:
I'm not familiar with this plugin.  Why have you added it, and what does it offer over the `jquery` plugin you use below?